### PR TITLE
[4.1.0] Add Istio service to service sample

### PR DESCRIPTION
**Pull request by renuka-fernando**
## Purpose
subject

### Screenshot of the page
image

### CODE REVIEW

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/service-mesh/istio.md [hunk 1]
```diff
@@ -20,3 +20,11 @@ Microservices can be deployed in the Istio service mesh and made available using
 
 
 Istio Sidecar sample can be found in [GitHub](https://github.com/wso2/product-microgateway/tree/main/samples/istio/sidecar-mode).
+
+### Service to Service Communication
+
+<img src="{{base_path}}/assets/img/deploy/istio-east-west-traffic.png" width="75%" alt="East-West traffic API management">
+
+Microservices can be deployed in the Istio service mesh and the communication between microservices can be configured to route through Choreo Connect as an Internal API Gateway. Choreo Connect is deployed in the Istio service mesh as a separate service where it applies API Management for the microservices. Requests arrive Choreo Connect validated before passing it the relevant microservice , as indicated in the diagram.
+
```
Comment (pubudu538) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/service-mesh/istio.md:29]: Shall we add a sentence saying CC can be acted as the edge gateway as well as an internal gateway in this case?

Comment (renuka-fernando) [reply en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/service-mesh/istio.md:29]: Added with https://github.com/wso2/docs-apim/pull/5986/commits/7e2b9a372384951593e222d85f22a60c7ee764d8

**Review by pubudu538 — APPROVED** [review]

**Comment by renuka-fernando** [comment]
Screenshot of the page
image

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/service-mesh/istio.md [hunk 2]
```diff
@@ -20,3 +20,11 @@ Microservices can be deployed in the Istio service mesh and made available using
 
 
 Istio Sidecar sample can be found in [GitHub](https://github.com/wso2/product-microgateway/tree/main/samples/istio/sidecar-mode).
+
+### Service to Service Communication
+
+<img src="{{base_path}}/assets/img/deploy/istio-east-west-traffic.png" width="75%" alt="East-West traffic API management">
+
+Microservices can be deployed in the Istio service mesh and the communication between microservices can be configured to route through Choreo Connect as an Internal API Gateway. Choreo Connect is deployed in the Istio service mesh as a separate service where it applies API Management for the microservices. Requests arrive Choreo Connect validated before passing it the relevant microservice, as indicated in the diagram. In this scenario Choreo Connect act as the edge gateway as well as the internal gateway.
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/service-mesh/istio.md:28]:
```suggestion
Microservices can be deployed in the Istio service mesh and the communication between microservices can be configured to route through Choreo Connect as an Internal API Gateway. Choreo Connect is deployed in the Istio service mesh as a separate service where it applies API Management for the microservices. Requests arrive that are validated in Choreo Connect before passing it the relevant microservice, as indicated in the diagram. In this scenario Choreo Connect acts as the edge Gateway as well as the internal Gateway.
```

**Review by samgnaniah — APPROVED** [review]

**Pull request merged**